src/client/mod.rs

```diff
@@ -439,7 +439,7 @@ impl KafkaClient {
     ///     let connector = builder.build();
     ///
     ///     let mut client = KafkaClient::new_secure(vec!("localhost:9092".to_owned()),
-    ///                                              SecurityConfig::new(connector));
+    ///                                              SecurityConfig::new(connector, true));
```

Comment:
see my comment below on the `with_hostname_verification` suggestion and please do not suggest its usage in this code example. however, at the same time please update the "see" section in this rustdoc comment (at its end): i think `openssl_verify` is not worth mentioning anymore, but a "see also `SecurityConfig#with_hostname_verification`" hint would be surely helpful.